docs/tg/xilinx.md

```diff
@@ -7,7 +7,7 @@ This page describes how to map Cheshire on Xilinx FPGAs to *execute baremetal pr
 We currently provide working setups for:
 
 - Digilent Genesys 2 with Vivado `>= 2020.2`
-- Xilinx VCU128 with Vivado `>= 2020.2`
+- Xilinx VCU128/VCU118 with Vivado `>= 2020.2`
```

Comment:
Better to put this on its own line for legibility

Reply:
Should be fixed